src/main/java/org/ow2/proactive/catalog/rest/controller/CatalogObjectController.java

```diff
@@ -190,6 +190,13 @@ public Set<String> listContentTypes() {
         return catalogObjectService.getContentTypes();
     }
 
+    @ApiOperation(value = "Lists all workflow tags for all objects")
```

Comment:
Lists all tags for all objects

and the request should be /tags

Maybe tags will not only be available for workflow objects